tests/cancun/eip4844_blobs/test_excess_blob_gas_fork_transition.py

```diff
@@ -211,28 +297,31 @@ def test_invalid_post_fork_block_without_blob_fields(
             )
         ],
         genesis_environment=env,
-        tag="blob_fields_missing_post_fork",
     )
 
 
-@pytest.mark.parametrize(
-    "post_fork_block_count,blob_count_per_block",
-    [
-        (
-            SpecHelpers.get_min_excess_blobs_for_blob_gas_price(fork=Cancun, blob_gas_price=2)
-            // (Cancun.max_blobs_per_block() - Cancun.target_blobs_per_block())
+@pytest.mark.valid_at_transition_to("Cancun", subsequent_forks=True)
+@pytest.mark.parametrize_by_fork(
+    "post_fork_block_count,post_fork_blobs_per_block",
+    lambda fork: [
+        pytest.param(
+            SpecHelpers.get_min_excess_blobs_for_blob_gas_price(fork=fork, blob_gas_price=2)
+            // (
+                fork.max_blobs_per_block(timestamp=FORK_TIMESTAMP)
+                - fork.target_blobs_per_block(timestamp=FORK_TIMESTAMP)
+            )
             + 2,
-            Cancun.max_blobs_per_block(),
+            fork.max_blobs_per_block(timestamp=FORK_TIMESTAMP),
+            id="max_blobs",
         ),
-        (10, 0),
-        (10, Cancun.target_blobs_per_block()),
+        pytest.param(10, 0, id="no_blobs"),
```

Comment:
Loving `parametrize_by_fork`. Checked the fixtures by hand; max/target blob configs are respected.

This test case in L317 tests:
1. For `ShanghaiToCancunAtTime15k`: "0 blobs pre-fork to 0 blobs post-fork"
2. For `CancunAtPragueAtTime15k`: "max blobs pre-fork to 0 blobs post-fork"

2. is clearly a special case, which could deserve more exploration and variations, I added some in #1107.